galaxy_ng/tests/integration/api/test_rbac_roles.py

```diff
@@ -0,0 +1,316 @@
+    index_exec_env,
+)
+
+log = logging.getLogger(__name__)
+
+
+GLOBAL_ACTIONS = [
+    view_groups,
+    delete_groups,
+    add_groups,
+    change_groups,
+    view_users,
+    delete_users,
+    add_users,
+    change_users,
+    view_role,
+    delete_role,
+    add_role,
+    change_role,
+    view_tasks,
+    create_collection_namespace,
+    # create_collection_namespace_object,
+    change_collection_namespace,
+    delete_collection_namespace,
+    upload_collection_to_namespace,
+    delete_collection,
+    configure_collection_sync,
+    launch_collection_sync,
+    view_sync_configuration,
+    approve_collections,
+    reject_collections,
+    deprecate_collections,
+    # deprecate_collections_object,
+    undeprecate_collections,
+    # undeprecate_collections_object,
+    upload_collection_to_namespace,
+    # upload_collection_to_namespace_object,
+    create_container_registry_remote,
+    change_container_registry_remote,
+    delete_container_registry_remote,
+    create_exec_env,
+    delete_exec_env,
+    change_exec_env_desc,
+    change_exec_env_readme,
+    index_exec_env,
+]
+# OBJECT_ACTIONS = [
+#     create_collection_namespace_object,
+#     upload_collection_to_namespace_object,
+#     deprecate_collections_object,
+#     undeprecate_collections_object,
+# ]
+ROLES_TO_TEST = {
+    "galaxy.content_admin": {
+        view_tasks,
+        view_groups,
+        create_collection_namespace,
+        change_collection_namespace,
+        delete_collection_namespace,
+        upload_collection_to_namespace,
+        delete_collection,
+        configure_collection_sync,
+        launch_collection_sync,
+        view_sync_configuration,
+        approve_collections,
+        reject_collections,
+        deprecate_collections,
+        undeprecate_collections,
+        create_exec_env,
+        delete_exec_env,
+        change_exec_env_desc,
+        change_exec_env_readme,
+        create_container_registry_remote,
+        change_container_registry_remote,
+        delete_container_registry_remote,
+        index_exec_env,
+        # create_containers_under_existing_container_namespace,
+        # push_containers_to_existing_container_namespace,
+        # change_container_namespace,
+        # change_container_namespace_object,
+        # tag_untag_container_namespace,
+        # sync_remote_container,
+        # create_remote_container,
+    },
+    "galaxy.collection_admin": {
+        view_tasks,
+        view_groups,
+        create_collection_namespace,
+        change_collection_namespace,
+        delete_collection_namespace,
+        upload_collection_to_namespace,
+        delete_collection,
+        configure_collection_sync,
+        launch_collection_sync,
+        view_sync_configuration,
+        approve_collections,
+        reject_collections,
+        deprecate_collections,
+        undeprecate_collections,
+    },
+    "galaxy.collection_publisher": {
+        view_tasks,  # error?
```

Comment:
Have you done any debugging to figure out why the actions that are failing are failing? Right now all the tests are failing and some actions seem to be failing across the board. I'd like to know which are failing and if the tests are broken or the app is broken.